types/react-data-grid/index.d.ts

```diff
@@ -219,6 +219,22 @@ declare namespace AdazzleReactDataGrid {
          * @param columnKey the column key that we are looking to pull values from
          */
         getValidFilterValues?: (columnKey: string) => Array<any>
+
+        getCellActions?: (column: Column, row: object) => (ActionButton | ActionMenu)[]
+    }
+
+    type ActionButton = {
+        icon: string;
+        callback: Function;
```

Comment:
Looks good, but could you define the callbacks as `() => void` instead of Function?

Reply:
Done!

Reply:
The one on line 228 as well, please

Reply:
![](http://i0.kym-cdn.com/entries/icons/mobile/000/000/554/picard-facepalm.jpg)

Also done!